README.md

```diff
@@ -68,6 +74,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+A: We have to call it inside the method analyze_mood before we do the word by word comparison so that it can accurately compare words without the punctuation.
```

Comment:
We don't _have_ to call it inside analze_mood, though I agree that's the correct place to do so. Why do you think that's the case?